lang/en.json

```diff
@@ -674,7 +674,9 @@
         "choose_an_option": "Please choose an option",
         "select_one": "Please select one",
         "description": "Description",
-        "retail_price": "MSRP",
+        "retail_price": "MSRP:",
+        "price_was": "Was:",
+        "price_now": "Now:",
```

Comment:
🍹 in my opinion we would leave the `:` out of the lang file and in the template directly, @junedkazi do you have an opinion?

Reply:
i think you could argue the colon is part of the language.  we also have it in MSRP: above so i think think it's fine to stay

Reply:
thank you